third_party/terraform/website/docs/r/sql_user.html.markdown

```diff
@@ -53,6 +53,12 @@ The following arguments are supported:
 * `password` - (Optional) The password for the user. Can be updated. For Postgres
     instances this is a Required field.
 
+* `deletion_policy` - (Optional) The deletion policy for the user.
+    Setting ABANDON allows the resource to be abandoned rather than deleted. This is useful
+    for situations where Postgres users that cannot be deleted due existing SQL permissions.
```

Comment:
This wording is a little funny. 
"This is useful for Postgres, where users cannot be deleted due to SQL permissions." Maybe?

Reply:
Hmmm, to me that sounds like the user cannot be deleted because Terraform doesn't have the right SQL permissions, which is not the case.

Maybe "This is useful for Postgres, where users cannot be deleted from the API if they have been granted SQL roles"?

Reply:
Sounds better to me in both cases!